__tests__/server.test.js

```diff
@@ -193,4 +193,100 @@ describe("/api/articles/:article_id/comments", () => {
             })
         })
     })
+    describe("POST", () => {
+        test("POST 201: Responds with 201 status code and the posted comment", () => {
+            const newComment = {
+                username: "lurker",
+                body: "Hello world"
+            }
+            return request(app)
+            .post("/api/articles/1/comments")
+            .send(newComment)
+            .expect(201)
+            .then(({body}) => {
+                expect(body.comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
```

Comment:
You can be more specific about the user and body here :)

Reply:
@Harrison365 
When you say user, do you mean the author when it comes back?

If so, would it just check those two, or would you check the whole object?